vendor/composer/autoload_static.php

```diff
@@ -61,6 +61,10 @@ class ComposerStaticInit22ddb69348c7ed922c96325249cef3d0
         array (
             'Psr\\Log\\' => 8,
```

Comment:
that is really odd. it should not be bringing in classes that do not exist (since you remove the phing directory). I wonder if that "global" thing is causing issues. Did you do the composer global install for phing? If so, rec removing the global install and then should work.